watertap/examples/chemistry/tests/test_water_softening.py

```diff
@@ -391,9 +391,6 @@ def test_stoich_inherent(self, water_stoich):
         orig_fixed_vars = fixed_variables_set(m)
         orig_act_consts = activated_constraints_set(m)
 
-        solver.options['bound_push'] = 1e-10
-        solver.options['mu_init'] = 1e-6
-        solver.options['nlp_scaling_method'] = 'user-scaling'
```

Comment:
Out of curiosity, did these tests fail when keeping these now deleted options in? Or did they still pass but pass more slowly than when they were cleared away?

Reply:
At this point I would need to check. I probably removed them up-front because one of the points of this change is to not need these options anymore.

Note that all the various flowsheet-tests in `examples` still pass, even though they largely also use similar options.